source/includes/open-new-connection.rst

```diff
@@ -0,0 +1,3 @@
+In the bottom panel of the :guilabel:`Connections Sidebar`,
+click the :icon-lg:`Plus` button to open the :guilabel:`New Connection`
```

Comment:
[nit] tiny nit on wording, but I think we want to refer to the plus as an "icon" since a button usually indicates the presence of a border or contrasting UI color.
```suggestion
click the :icon-lg:`Plus` icon to open the :guilabel:`New Connection`
```

Reply:
Fixed!